### ISSUE TITLE
Error in "yarn start" step

### ISSUE THREAD

**Issue by sorny92** [lazd/coronadatascraper#21]
Ok,
I'm starting with the scraper and I'm finding myself with problems setting the invironment.
I'm running ```Ubuntu 18.04```
node version: ```v12.16.1```
yarn version: ```1.22.4```

So far I've run:
```bash
yarn install
yarn start
```
I got an error related to some node arguments but I updated node to the version I said in the top and there wasn't error anymore.
When I run: ```yarn start``` now I get the next error:
```bash
yarn run v1.22.4
$ NODE_OPTIONS='--insecure-http-parser' node index.js
internal/modules/cjs/loader.js:1174
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
      ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /home/esteve/projects/coronadatascraper/index.js
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1174:13)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  code: 'ERR_REQUIRE_ESM'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I have not that much experience in JS development, I'm more of  a Python guy myself :disappointed:

_Originally posted by @sorny92 in https://github.com/lazd/coronadatascraper/issues/14#issuecomment-599113167_

**Comment by sorny92** [comment]
I solved it updating nodejs to v13.11.0

**Comment by sorny92** [comment]
I solved it updating nodejs to v13.11.0

**Comment by lazd** [comment]
Oh nice one. Note you also can't use `require` anywhere, you have to use ES6-style `import`.
